daily_milestone.go

```diff
@@ -121,7 +138,7 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 func getMilestones(baseURL string, token string, projectID string) ([]simpleMilestone, error) {
 	milestones := []milestoneAPI{}
 	list := []simpleMilestone{}
-	strURL := []string{baseURL, projectID, "/milestones"}
+	strURL := []string{baseURL, projectID, "/milestones?state=active&per_page=100"}
```

Comment:
The change now filters open/closed on the server and gives back 100 results (the maximum possible per page).